README.md

```diff
@@ -167,7 +174,8 @@ Happy querying!
 - `engine (Engine)`: a Tartiflette [engine](https://tartiflette.io/docs/api/engine). Required if `sdl` is not given.
 - `sdl (str)`: a GraphQL schema defined using the [GraphQL Schema Definition Language](https://graphql.org/learn/schema/). Required if `engine` is not given.
 - `graphiql (bool)`: whether to serve the GraphiQL when accessing the endpoint via a web browser. Defaults to `True`.
-- `path (str)`: the path which clients should make GraphQL queries to. Defaults to `"/"`. A popular alternative is `"/graphql"`.
+- `path (str)`: the path which clients should make GraphQL queries to. Defaults to `"/graphql"`.
+- `root (bool)`: shortcut for `path=""`. Defaults to `False`.
```

Comment:
I'm wondering why this parameter exists?

If the developers want to expose tartiflette to another endpoint, they will tweak the `path` parameter to another value than the default one, Am I right?

I think that `path` is enough.

Reply:
> I'm wondering why this parameter exists?

Mostly because I believe writing `graphql = TartifletteApp(sdl=sdl, path="")` is IMO quite ugly, but I'm okay with removing `root` as it does break the principle of "there should be one obvious way to do it".

What still bugs me is that we need to modify the app in `graphql.py` in order to mount it *at the same URL path* in `app.py`.

After thinking about it, it's probably best to set the default `path` to `""`, as initially suggested in the issue.
This way serving as a standalone app doesn't require accessing a specific endpoint (which makes sense), and we can explicitly mount at `/graphql` when mounting on another ASGI app.
And this solves the problem of editing multiple files just to mount the app.

Thoughts?

Reply:
Ok, thus, it should be interesting to add a code sample at the end of the README.me which explain `How to plug TartifletteApp to an existing Starlette application?`. What do you think?